ColorPatterns/Helpers/Starling.swift

```diff
@@ -0,0 +1,266 @@
+//
+//  Starling
+//
+//  Released under: MIT License
+//  Copyright (c) 2018 by Matt Reagan
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+
+import Foundation
+import AVFoundation
+
+/// Typealias used for identifying specific sound effects
+public typealias SoundIdentifier = String
+
+/// Errors specific to Starling's loading or playback functions
+enum StarlingError: Error {
+    case resourceNotFound(name: String)
+    case invalidSoundIdentifier(name: String)
+    case audioLoadingFailure
+}
+
+public class Starling {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/8c9d8b0cbd258e51fec07cc9cd8e5c65cf9e26ea/ColorPatterns/Helpers/Starling.swift#L38"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>